ci/do_ci.sh

```diff
@@ -41,7 +41,7 @@ function do_clang_tidy() {
 
 function do_unit_test_coverage() {
     export TEST_TARGETS="//test/... -//test:python_test"
-    export COVERAGE_THRESHOLD=94.2
+    export COVERAGE_THRESHOLD=93.0
```

Comment:
This seems like a rather large decrease, looking at the coverage report we could set this set this at `93.9` with the current state.

Reply:
having said that, looking at https://26453-180498819-gh.circle-artifacts.com/0/coverage/source/client/output_formatter_impl.cc.gcov.html, I think it should be possible to hit some of the failure cases for the Fortio output formatter, by creating a test that provides an input that doesn't include a "global" result. As far as I am concerned this can be moved to a new issue though.